xmppserver/src/main/java/org/jivesoftware/openfire/ldap/LdapManager.java

```diff
@@ -1914,20 +1882,19 @@ public void setGroupSearchFilter(String groupSearchFilter) {
         properties.put("ldap.groupSearchFilter", groupSearchFilter);
     }
 
+    /**
+     * @Deprecated As of Openfire 4.5.0, DNs used by Openfire are never enclosed.
+     */
+    @Deprecated
     public boolean isEnclosingDNs() {
-        String encloseStr = properties.get("ldap.encloseDNs");
-        if (encloseStr != null) {
-            encloseDNs = Boolean.valueOf(encloseStr);
-        } else {
-            encloseDNs = true;
-        }
-
-        return encloseDNs;
+        return false;
     }
 
+    /**
+     * @Deprecated As of Openfire 4.5.0, DNs used by Openfire are never enclosed.
+     */
+    @Deprecated
     public void setIsEnclosingDNs(boolean enable) {
```

Comment:
+1 for keeping old APIs for backward compatability. 

But, enough public APIs have changed (returning LdapName instead of String, taking an LdapName instead of a String parameter) that backward compatability is lost anywau.

Either
a) Add new methods, keeping the old ones, or
b) Remove this old method too.